### PR TITLE
Added Elasticsearch5 dependency and output modules

### PR DESCRIPTION
**Pull request by Onager**
Added Elasticsearch5 dependency and output modules

### CODE REVIEW

File: plaso/dependencies.py [hunk 1]
```diff
@@ -32,7 +32,8 @@
     'dfwinreg': ('__version__', '20180712', None, True),
     'dtfabric': ('__version__', '20180707', None, True),
     'efilter': ('', '1.5', None, True),
-    'elasticsearch': ('__versionstr__', '5.4.0', '5.5.1', True),
+    'elasticsearch': ('__versionstr__', '5.4.0', None, False),
+    'elasticsearch5': ('__versionstr__', '', None, True),
```
Comment (joachimmetz) [inline plaso/dependencies.py]: Should 5 also be at least 5.4.0?

File: plaso/dependencies.py [hunk 2]
```diff
@@ -32,7 +32,8 @@
     'dfwinreg': ('__version__', '20180712', None, True),
     'dtfabric': ('__version__', '20180707', None, True),
     'efilter': ('', '1.5', None, True),
-    'elasticsearch': ('__versionstr__', '5.4.0', '5.5.1', True),
+    'elasticsearch': ('__versionstr__', '5.4.0', None, False),
```
Comment (joachimmetz) [inline plaso/dependencies.py]: What are your thoughts to make this minimum version 6?

File: plaso/output/elastic.py [hunk 3]
```diff
@@ -72,5 +72,72 @@ def WriteHeader(self):
...
+              }
+          }
+      }]
+
+    self._Connect()
+
+    self._CreateIndexIfNotExists(self._index_name, mappings)
+
+
 manager.OutputManager.RegisterOutput(
```
Comment (joachimmetz) [inline plaso/output/elastic.py]: what are your thoughts to enable this when elasticsearch > 6?

**Comment by joachimmetz** [comment]
@Onager please resolve "This branch has conflicts that must be resolved" and roll out elasticsearch5 module to dev binary repos

File: plaso/dependencies.py (hunk 2 above)
Comment (Onager) [reply plaso/dependencies.py]: I think that's a good idea - updated

File: plaso/dependencies.py (hunk 1 above)
Comment (Onager) [reply plaso/dependencies.py]: Indeed it should

File: plaso/output/elastic.py (hunk 3 above)
Comment (Onager) [reply plaso/output/elastic.py]: I'm not quite sure what you meant here, but I've changed the elasticsearch module requirement to be >6

**Pull request merged**